### ISSUE TITLE
[Request] Betterpushback volume control to be removed from radio slider

### ISSUE THREAD

**Issue by thagreenkronic** [skiselkov/BetterPushbackC#239]
Hi, for those of us who use an ATC plugin or online ATC we need to turn down radio sounds otherwise 2 ATIS's will play over the radio.
Many betterpushback sounds are linked to that slider, and turning them down is not an option for many of us since your plugin is so good ;)

So either we have to put up with 2 ATIS playing over the radio at the same time or not hear the wonderful betterpushback sounds.
Would it be possible to link the betterpushback sounds to the master volume of the game? Or maybe have it own slider within the plugin menu?

**Comment by nguenther** [comment]
I miss this option, too!

**Comment by gkkes** [comment]
I agree. In trying to find an acceptable level I find that the female advisor voice is at a comfortable level, but the Tug Driver is too high in the radio level mix. It appears there is no control over the level of the advisor as she appears independent of the xplane volume mixer.
Love the app, Thanks
